src/pilot/assign_role.py

```diff
@@ -321,11 +323,13 @@ def define_compute_logical_disks(drac_client, raid_controller_name):
     raid_10_logical_disk = define_single_raid_10_logical_disk(
         drac_client, raid_controller_name)
 
+    logical_disks = list()
     if raid_10_logical_disk is None:
         return None
```

Comment:
The above line looks good.  Please test as noted above, but passing a role of "compute" instead.